### PR TITLE
feature/mx_1517 add mapping connector

### PR DESCRIPTION
**Pull request by erichesse**
# PR Context

# Added

- add mapping connector

# Changes

# Deprecated

# Removed

# Fixed

# Security

### CODE REVIEW

File: mex/mapping/connector.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import yaml
+from pydantic import BaseModel
+
+from mex.common.models.mapping import MAPPING_MODEL_BY_EXTRACTED_CLASS_NAME
+from mex.common.types import AssetsPath
+
+
+def get_mapping_model(path: AssetsPath, model_type: type[BaseModel]) -> BaseModel:
```
Comment (cutoffthetop) [inline mex/mapping/connector.py:8]: hm, this is rather an `extract` function than a connector, wouldn't you say? i know, the ticket is phrased differently but let's disregard that 😅

File: assets/mappings/__final__/test_mapping/access-platform.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    mappingRules:
+    - rule: "Assign 'stable target id' of primary source with identifier 'nokeda' in /raw-data/primary-sources/primary-sources.json."
+identifier:
+  - fieldInPrimarySource: n/a
+    mappingRules:
+    - rule: Assign identifier.
+stableTargetId:
+  - fieldInPrimarySource: n/a
+    mappingRules:
+    - rule: Assign 'stable target id'  of merged item.
```
Comment (cutoffthetop) [inline assets/mappings/__final__/test_mapping/access-platform.yaml:43]: missing a newline here

**Review by cutoffthetop — CHANGES_REQUESTED** [review]
looks good 👍 i have just one naming-things-issue

File: mex/mapping/connector.py (hunk 1 above)
Comment (erichesse) [reply mex/mapping/connector.py]: done

File: assets/mappings/__final__/test_mapping/access-platform.yaml (hunk 2 above)
Comment (erichesse) [reply assets/mappings/__final__/test_mapping/access-platform.yaml]: done

File: .pre-commit-config.yaml.rej [hunk 3]
```diff
@@ -0,0 +1,16 @@
+diff a/.pre-commit-config.yaml b/.pre-commit-config.yaml	(rejected hunks)
```
Comment (cutoffthetop) [inline .pre-commit-config.yaml.rej:1]: `.rej` files should not be commited, they are produced by cruft to show you the diff between the current repo and the cookiecutter template repo for changes that could not automatically be applied

File: pyproject.toml.rej [hunk 4]
```diff
@@ -0,0 +1,17 @@
+diff a/pyproject.toml b/pyproject.toml	(rejected hunks)
```
Comment (cutoffthetop) [inline pyproject.toml.rej:1]: see above

**Review by cutoffthetop — APPROVED** [review]

**Pull request merged**